src/styles/typography.scss

```diff
@@ -4,13 +4,13 @@
   // Sizes
   --text-size-0: 12px;
   --text-size-1: 14px;
-  --text-size-2: var(--base-font-size);                      // 16px
+  --text-size-2: var(--base-font-size); // 16px
   --text-size-3: calc(var(--base-font-size) * var(--scale)); // 20px
-  --text-size-4: calc(var(--text-size-3) * var(--scale));    // 25px
-  --text-size-5: calc(var(--text-size-4) * var(--scale));    // 31px
-  --text-size-6: calc(var(--text-size-5) * var(--scale));    // 39px
-  --text-size-7: calc(var(--text-size-6) * var(--scale));    // 48px
-  --text-size-8: calc(var(--text-size-7) * var(--scale));    // 61px
+  --text-size-4: calc(var(--text-size-3) * var(--scale)); // 25px
```

Comment:
Oh my! Can you check your linter config please 👁️

Reply:
i'll take a look

Reply:
run `yarn prettier . --write` you will see that this file will change. "In VIM we trust" 😅 .